docs/Explorer.md

```diff
@@ -8,6 +8,24 @@ The Kafka explorer shows configured clusters with their topics, brokers, consume
 
 ![Screenshot-3](assets/screen-3.png)
 
+## Actions
+
+### Copy Label
+
+You can copy label of any tree node in the clipboard with `Keyboard` or `"Copy Label" contextual menu`:

```

Comment:
You can copy the label of any tree node in the clipboard with `Ctrl+C` (`Cmd+C` on Mac) or  via the `Copy Label` contextual menu:

Reply:
fixed